docs/release/3.4/release-3-4-44-2.md

```diff
@@ -4,17 +4,13 @@ OSG Data Release 3.4.44-2
 **Release Date:** 2020-03-04    
 **Supported OS Versions:** EL7, EL6
 
-!!!tip "Want faster access to production-ready software?"
-    OSG 3.5 and 3.4 offer a rolling release repository where packages are added as soon as they pass acceptance testing.
-    To install packages from this repository, enable `[osg-rolling]` in `/etc/yum.repos.d/osg-rolling.repo`:
-
-        [osg-rolling]
-        ...
-        enabled=1
-
-    Or for one-time installations, append the following to your `yum` command:
-
-        --enablerepo=osg-rolling
+!!!warning "OSG 3.4 End-of-Life Approaching"
+    According to our
+    [OSG Software Release Series Support Policy](https://opensciencegrid.org/technology/policy/release-series/),
+    the OSG 3.4 series is due to reach
+    [end-of-life](https://opensciencegrid.org/technology/policy/release-series/#life-cycle-dates) in **November 2020**.
```

Comment:
I think the release series support page is short enough that we don't need the second link

Reply:
I did that because on my laptop, the screen ends at the end of text. And it is the very final sounding statement. If you don't happen to look at the scroll bar, you might not even notice the table at the bottom.

Reply:
Fair enough!